### PR TITLE
[ti_ticura] Download feed via mode=url direct URL instead of following a redirect

### PR DESCRIPTION
**Pull request by eduardenes-ticura**
## Proposed commit message

The `indicator` data stream's feed endpoint 307-redirects to a time-limited
download URL. For large feeds (hundreds of MB), the CEL input auto-following that
redirect failed with `http: read on closed response body` — the agent went
degraded and ingested nothing (a direct, non-redirected download of the same body
works fine).

This requests the feed with `mode=url`, so the API returns
`200 {"downloadUrl": "..."}` instead of a 307. The CEL program now resolves that
URL (authenticated with the API key) and downloads the feed directly from it — no
redirect. The download URL is short-lived and never persisted, and the API key is
sent only to the API endpoint. The `Ticura URL` variable is now hidden
(`show_user: false`) since the endpoint is fixed, and the related README docs were
removed.

## Checklist

- [x] I have reviewed [tips for building integrations](https://www.elastic.co/docs/extend/integrations/tips-for-building) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).
- [x] I have verified that any added dashboard complies with Kibana's [Dashboard good practices](https://docs.elastic.dev/ux-guidelines/data-viz/dashboard-good-practices)

## How to test this PR locally

Add the **Ticura** integration with a valid API key + an agent; confirm indicators
ingest into `logs-ti_ticura.indicator-*` with the agent healthy. `elastic-package test`
(pipeline, asset, static, policy, system) all pass.

## Related issues

-

### CODE REVIEW

**Comment by eduardenes-ticura** [comment]
@kcreddy we just updated our package to handle an edge case with very big feeds. could you help another time to review , trigger test and release?

**Comment by kcreddy** [comment]
@vera-review-bot review

File: packages/ti_ticura/data_stream/indicator/_dev/test/policy/test-all.expected [hunk 1]
```diff
@@ -18,28 +18,52 @@ inputs:
...
+                        .map(l, {"message": l.trim_space()}),
+                      "want_more": false,
+                    }
+                  :
+                    {
+                      "events": {
+                        "error": {
+                          "code": string(resp.StatusCode),
+                          "id": string(resp.Status),
+                          "message": "GET (feed download): " + (
```
Comment (kcreddy) [inline packages/ti_ticura/data_stream/indicator/_dev/test/policy/test-all.expected]: Lets add the downloadUrl to the error as well, it helps in debugging what exactly failed.

File: packages/ti_ticura/data_stream/indicator/agent/stream/cel.yml.hbs [hunk 2]
```diff
@@ -35,28 +34,52 @@ redact:
...
       }
-    }).do_request().as(resp, resp.StatusCode == 200 ?
-      {
-        "events": string(resp.Body).split("\n")
-          .filter(l, size(l) > 0)
-          .map(l, {"message": l.trim_space()}),
-        "want_more": false,
-      }
+    }).do_request().as(meta, meta.StatusCode == 200 ?
+      bytes(meta.Body).decode_json().as(m, has(m.downloadUrl) ?
```
Comment (kcreddy) [inline packages/ti_ticura/data_stream/indicator/agent/stream/cel.yml.hbs]:
```suggestion
      meta.Body.decode_json().as(m, has(m.downloadUrl) ?
```

No need to convert to `bytes` after `8.15`+ stack.

**Comment by eduardenes-ticura** [comment]
@kcreddy thank you for reviewing. fixed as requested.

File: packages/ti_ticura/data_stream/indicator/manifest.yml [hunk 3]
```diff
@@ -9,10 +9,10 @@ streams:
         title: Ticura URL
         multi: false
         required: true
-        show_user: true
-        default: https://api.ticura.io/v1/feeds/download/full?targetFormat=ECSV1
+        show_user: false
```
[the remark this answers is not in the document]
Comment (eduardenes-ticura) [reply packages/ti_ticura/data_stream/indicator/manifest.yml:12]: fixed

**Review by kcreddy — APPROVED** [review]
